nrich-search/README.md

```diff
@@ -151,14 +180,11 @@ public class CarSearchResult {
     @Projection(path = "carType.model")
     private final String carTypeModel;
 }
-
-
 ```
 
-Then in our service we would invoke it like this:
+After search and result class was defined, the service that actually executes the query can be written like this:
```

Comment:
`is defined` sounds better in this context.

Reply:
Maybe even `are defined` because it references a plural.